buildSrc/src/main/groovy/bio.terra.policy.java-common-conventions.gradle

```diff
@@ -44,24 +44,19 @@ dependencies {
     compileOnly "com.google.code.findbugs:annotations:3.0.1"
     implementation 'org.slf4j:slf4j-api:2.0.7'
 
-    testImplementation 'ch.qos.logback:logback-classic:1.4.14'
-    testImplementation 'ch.qos.logback:logback-core:1.4.14'
+    testImplementation 'ch.qos.logback:logback-classic:1.5.18'
+    testImplementation 'ch.qos.logback:logback-core:1.5.18'
     testImplementation 'org.hamcrest:hamcrest:2.2'
 
     implementation 'io.swagger.core.v3:swagger-annotations:2.2.0'
 
     swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli:3.0.47'
 
-    var openTelemetryVersion = '1.31.0'
-    implementation "io.opentelemetry:opentelemetry-api:${openTelemetryVersion}"
-    implementation "io.opentelemetry:opentelemetry-sdk:${openTelemetryVersion}"
-    implementation "io.opentelemetry:opentelemetry-sdk-metrics:${openTelemetryVersion}"
-    implementation "io.opentelemetry:opentelemetry-exporter-logging:${openTelemetryVersion}"
-    implementation "io.opentelemetry.semconv:opentelemetry-semconv:1.21.0-alpha"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-webmvc-6.0:${openTelemetryVersion}-alpha"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:${openTelemetryVersion}"
-    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-boot:${openTelemetryVersion}-alpha"
-    implementation "com.google.cloud.opentelemetry:exporter-trace:0.25.2"
+    // these next two versions should match the versions used by terra-common-lib
+    implementation "io.opentelemetry:opentelemetry-api:1.42.1"
+    implementation "io.opentelemetry.instrumentation:opentelemetry-spring-boot-starter:2.8.0"
```

Comment:
I spent a few hours debugging OpenTelemetry version conflicts. I'd much rather simplify what TPS specifies for its dependencies by slimming down here.

Reply:
"should match versions used by terra-common-lib" seems like a tall order over any extended period of time